index.xml

```diff
@@ -220,6 +220,26 @@ both schemes.
 The following section details the Signature Parameters.
     </t>
 
+   <section anchor="v" title="v">
+     <t>
+OPTIONAL. The `v` field is a string denoting the implemented version
+of this specification. This enables the protocol to evolve, changing
+eg. the signature string specification or the supported algorithms and 
+parameters, in a backward-compatible way. The string value is "http-signatures-1".
+For backward compatibility, if this parameter is not present the Signature String 
+construction and the available parameters are left to the implementors. 
+
+Note: RFC EDITOR PLEASE DELETE THIS NOTE; The implementation of the final RFC 
+MUST use this context string, but implementations of drafts MUST NOT use it 
+and MUST use another draft-specific string beginning with "http-signatures-" instead,
+eg. "http-signatures-1-draft-11".
```

Comment:
This draft is already referred to in the [IETF tracker](https://datatracker.ietf.org/doc/draft-cavage-http-signatures/) by a distinct name per version, e.g.:
``draft-cavage-http-signatures-11``

Should we rather track that than invent a new scheme?

Reply:
fixed @liamdennehy